### DISCUSSION TITLE
Is graphql modules  production ready?

### DISCUSSION THREAD

**Discussion by muco-rolle** [Urigo/graphql-modules#1193] [Q&A] [answered]
Thanks for the work you have done and it's really awesome, but I see there's a lot changes I was wondering if it is stable enough to be used for a  production ready app

**Comment by maapteh** [comment, marked as answer]
We use it in production since the beginning, with high loads. Like every package before V1 it can change. But since V1 is coming (with some nice changes) and if you didn't use it yet, maybe best to wait for https://github.com/Urigo/graphql-modules/pull/1177 ? Personally i think the upgrade will not cost me much headache and v1 looks way more simple, but i can tell you when they released :)

**Comment by muco-rolle** [comment]
Can't wait V1

**Comment by russell-dot-js** [comment]
I've been using it in production for almost 2 years. No complaints, keeps our code clean

**Reply by sahanatroam** [reply]
@russell-dot-js keen to hear what sort of tests you've added to your codebase for confident deployments?
